tests/client_integration_test.go

```diff
@@ -855,7 +855,20 @@ func (s *clientIntegrationSuite) TestStickyAutoReset() {
 
 	// stop worker
 	s.worker.Stop()
-	time.Sleep(time.Second * 10) // wait 10s, after this time, matching will detect StickyWorkerUnavailable
```

Comment:
Is there any way we can shrink the timeout so we don't need to wait 11s in our test?

Reply:
Probably, but require more work. Test is in parallel so it does not mean we actually wait 10s here.